FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Extensions/MetricsLoggerExtensions.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.Health.Fhir.Synapse.Core.Extensions
 {
     public static class MetricsLoggerExtensions
     {
-        public static void LogTotalErrorsMetrics(this IMetricsLogger metricsLogger, Exception ex, string operationType, string message)
+        public static void LogTotalErrorsMetrics(this IMetricsLogger metricsLogger, Exception ex, string message, string operationType)
```

Comment:
how about change the type of operationType from string to `JobOperations`?

Reply:
JobOperation is not an enum object here. Change it to Enum is feasible but may be not for this bug fix.